content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,101 @@
+
+### Periodic Review
+
+It is vital to periodically review Kubernetes RBAC assignments for redundant entries, to avoid possible privilege escalation issues. If an attacker is able to create a user account with the same name as a deleted one, they will automatically inherit all the rights that that previous account had (where the right was assigned to the username specifically).
+
+## Kubernetes RBAC - Privilege Escalation Risks 
+
+Within Kubernetes RBAC there are a number of privileges which, if granted, can allow for a user or service account to escalate their rights in the cluster or affect systems outside the cluster.
+
+This document is intended to provide visibility of the areas where cluster administrators should take care, to ensure that they do not inadvertantly allow for more access to clusters than intended.
+
+### Secrets Objects - List
+
+Whilst it is generally clear that allowing **GET** access to secrets will allow a user to read their contents, which then allows for them to be re-used, it's important to note that **LIST** access effectively also allows for this, as when a list request is returned (e.g. via `kubectl get secrets -A -o yaml`) it will include the contents of the secret.
+
+### Workload Creation
+
+Users who are able to create workloads (e.g. pods) will be able to gain access to the underlying node if restrictions based on the [Pod Security Standards](/docs/concepts/security/pod-security-standards/) are not in place. Baseline or Restricted profiles should be used for all users with rights to create workloads in the cluster.
+
+Creating a workload in a namespace also grants indirect access to secrets in that namespace. Creating a pod in kube-system or a similarly privileged namespace can grant a user access to secrets they would not have through RBAC directly.  
+
+### Persistent Volume Creation
+
+As noted in the [Pod Security Policy](/docs/concepts/policy/pod-security-policy/#volumes-and-file-systems) documentation, access to create Persistent Volumes can allow for escalation of access to the underlying host.
+
+### Node/Proxy Access
+
+Users with access to the proxy sub-resource of node objects have rights to the Kubelet API, which allows for command execution on every pod on the node(s) which they have rights to. This access bypasses audit logging and admission control, so care should be taken before granting rights to this resource.
+
+### Escalate Right
+
+Generally the RBAC system prevents users from creating clusterroles with more rights than they possess. The exception to this is the escalate right. As noted in the [RBAC documentation](/docs/reference/access-authn-authz/rbac/#restrictions-on-role-creation-or-update) users with this right can effectively escalate their privileges.
```

Comment:
```suggestion
The RBAC system prevents users from creating ClusterRoles with more rights than they possess.
There is an explicit exception to that preventative mechanism: the `escalate` right. As noted in the [RBAC documentation](/docs/reference/access-authn-authz/rbac/#restrictions-on-role-creation-or-update) users with
this right can effectively escalate their privileges, because they are authorized to do so.
```

I would overall try to frame it that the `escalate` right is equivalent _within RBAC_ to full access.
We might want to hint that other mechanisms&mdash;a webhook authorizer, for example&mdash;can further
control a principal's ability to elevate their privileges.